CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to this repository
+
+## Contributions in English
+
+### Using Bold
+- to note step numbers. Example: __First__, __Second__, __Third__, etc
+- to designate important Creator Components, Objets, etc. Example: __Scene__, __Sprite Component__
+- to designate important Creator UI elements. Example: __Hierarchy Panel__, __Assets Panel__
+
+### Using Italics
+- list items. Example: *apple*, *orange*, *banana*, and *pear*
+
+### Using a single backtick
+- designate properties
```

Comment:
I think it's stricter: a line of code (or designate properties in code)

Reply:
As mentioned earlier I would like to add class name and function name here, and I agree with @jareguo , single line code can be wrapped in single backtick. e.g.

You can use `spriteFrame` property of `SpriteComponent` to get or set a `SpriteFrame` asset, like `let frame = sprite.spriteFrame`